pkg/segment/tracing/handler/tracehandler.go

```diff
@@ -365,3 +365,83 @@ func redMetricsToJson(redMetrics structs.RedMetrics, service string) ([]byte, er
 	result["p99"] = redMetrics.P99
 	return json.Marshal(result)
 }
+
+func DependencyGraphThread() {
+	time.Sleep(1 * time.Minute) // Wait for initial traces ingest first
+	for {
+		MakeTracesDependancyGraph()
+		time.Sleep(1 * time.Hour)
+	}
+}
+
+func MakeTracesDependancyGraph() {
+	nowTs := putils.GetCurrentTimeInMs()
+	startEpoch := nowTs - (60 * 60 * 1000)
```

Comment:
You're using a one-hour magic number here and above in `time.Sleep(1 * time.Hour)`. You should instead make a constant at the top of this file with a value of one hour and then read that constant for determining the `startEpoch` and how long to sleep. That way it's easier to change this later if we want to.